DEVELOPMENT.md

```diff
@@ -1,101 +1,39 @@
-
-```
-APOLLO_DIST_TAG=latest npm run release:start-ci-publish
-```
-
-#### Step 3b: Manually publishing
-
-__In the event that publishing via CI/CD is not possible, it may be done manually. Publishing manually should be avoided whenever possible.__
-
-1. Log into `npm` with the `apollo-bot` user.
-
-The `apollo-bot` user credentials are available to project owners, but generally used by CI/CD.
-Logging in with the following command will use a different npm user configuration file at `$HOME/.npmrc-apollo-bot` so as not to override personal login credentials which may already be used.
-
-```
-NPM_CONFIG_USERCONFIG="$HOME/.npmrc-apollo-bot" npm login
-```
-
-2. Publish using `lerna` and the `apollo-bot` credentials.
-
-
-> Note: By default, publishing will be done to the `latest` tag on npm.  To publish on a different `dist-tag` include the `--dist-tag` option below.  E.g. To publish to the `alpha` tag instead of `latest`, add `--dist-tag=alpha`.
-
-```
-DEBUG=lerna NPM_CONFIG_USERCONFIG="$HOME/.npmrc-apollo-bot" npx lerna publish from-git
-```
-
-### Step 4: Add the "Complete versioning details" links
-
-While it would be ideal to have this baked into the Git tag, it's not possible to do this in previous steps since we need to reference the commit hash that is created in Step 2's version bumping.
-
-Therefore, after publishing, add an entry to the CHANGELOG that was published which references the "Release" commit in order to provide easier visibility into what was in that that release.  It should be in the following format, where `COMMIT_HASH` is replaced with the Git reference for the most `Release` commit found in `git log` and should immediately follow the `### vX.Y.Z` header of the version just published:
-
-```
-> [See complete versioning details.](https://github.com/apollographql/apollo-server/commit/COMMIT_HASH)
-```
-
-### Step 5: Update GitHub Milestones
-
-The milestone for the release should be finalized.  If it didn't have a concrete version as its name, it should be renamed to the version that was finally released.
-
-Then, any remaining issues or PRs which did not land in this version should be moved to a newly-created milestone which reflects the newly intended release for them.
-
-Finally, _close_ (don't delete) the previous milestone once it is at 100% completion.
+* Each dual-published package has 2 tsconfigs: one for ESM and one for CJS. These must also be referenced at the top-level in the respective esm/cjs-specific tsconfig files.
+* Deep imports in the `@apollo/server` package must be listed within the `package.json` "exports" property. It's important to note that each "exports" entry must be specifically in the order: "types", "import", "require". [Typescript docs for reference](https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing)
```

Comment:
probably say that this is used by Node (right?) and tsc with moduleResolution whatever it is